next.config.mjs

```diff
@@ -1,16 +1,19 @@
 /** @type {import('next').NextConfig} */
-import * as path from 'path';
+import * as path from 'path'
 
 const nextConfig = {
   reactStrictMode: true,
   output: 'standalone',
   webpack(config) {
-    config.module.rules[2].oneOf?.forEach((one) => {
-      if (!`${one.issuer?.and}`.includes('_app')) return;
-        one.issuer.and = [path.resolve()];
-    });
-    return config;
+    config.module.rules[2].oneOf?.forEach(one => {
+      if (!`${one.issuer?.and}`.includes('_app')) return
+      one.issuer.and = [path.resolve()]
+    })
+    return config
   },
+  images: {
+    domains: ['images.unsplash.com']
```

Comment:
What this means?

Reply:
If we don't specify this domain, we can't load all the `images.unsplash.com` images: `https://nextjs.org/docs/messages/next-image-unconfigured-host`

Reply:
I thought, all the images received will come from an okp4 domain. I'll patch it with the wildcard, good to know.